app/views/rave/index.rss.builder

```diff
@@ -0,0 +1,32 @@
+# basic format copied from http://www.getrave.com/rss/chapman/channel1
+xml.instruct! :xml, :version => "1.0"
+xml.rss :version => "2.0" do
+  xml.channel do
+    xml.title "Chapman Panther Alerts"
+    xml.description "Welcome to Panther-Alert, Chapman University's Emergency Notification system.  Panther-Alert sends emergency messages to your mobile or fixed device of choice so you get emergency messages quickly wherever you are."
+    xml.link "http://www.getrave.com/rss/chapman/channel1"
+    xml.category "Panther Alerts"
+
+    xml.image do
+      xml.title "Chapman Panther Alerts"
+      xml.url "http://www.chapman.edu/campus-services/public-safety/_files/Panther%20Alert%20Logo-Web.jpg"
+      xml.link "http://www.getrave.com/rss/chapman/channel1"
+    end
+
+    xml.item do
+
+      if @status == "on"
```

Comment:
It was my understanding that the setup right now on http://www.getrave.com/rss/chapman/channel1 is the way it will look for non-emergencies, and I have not yet received confirmation on how the real emergency alerts will work. there is a comment at the top of this file that links to where I got the format from